### ISSUE TITLE
CustomElementRegistry not being wiped on hot reload

### ISSUE THREAD

**Issue by e111077** [stackblitz/core#416]
If you define a custom element using `customElement.define('my-element', MyElementDefinition)` upon hot reload, you get the error

```
Failed to execute 'define' on 'CustomElementRegistry': this name has already been used with this registry
Evaluating my-element.js
```

This is because custom element names must be singleton. Everything works if you just click on the reload button.

**Comment by seveves** [comment]
In what kind of project? Can you link to a project where we can easily reproduce this? Thanks! :)

**Comment by e111077** [comment]
Heya, [here is a repro ES6 project](https://stackblitz.com/edit/polymer-3-hello-world?file=my-element.js): you simply have to modify the `my-element.js` file.

**Comment by e111077** [comment]
fixed by [b2d397003ba393975abbca82fd8759d158b6e7f9](https://github.com/stackblitz/core/commit/b2d397003ba393975abbca82fd8759d158b6e7f9)
